website/docs/configure_hydra/Intro.md

```diff
@@ -68,3 +68,17 @@ You can see the full Hydra config using `--cfg hydra`:
 - *hydra.runtime.version*: Hydra's version
 - *hydra.runtime.cwd*: Original working directory the app was executed from
 
+## Hydra default OmegaConf resolvers
```

Comment:
```suggestion
## Hydra resolvers
```
1. Link to custom resolvers, no need for a link to interpolation as well.
2. Add a smalls section for each function, describing it.

### now
Create a string representing the current time using [strftime](https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior).
For example, for format the time you can use something like`${now:%H-%M-%S}`.